projects/can_slave/src/main.c

```diff
@@ -0,0 +1,58 @@
+#include <stdbool.h>
+#include "can_hw.h"
+#include "can_uart.h"
+#include "delay.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "uart.h"
+#include "wait.h"
+
+static UARTStorage s_uart_storage;
+
+static void prv_init_periph(void) {
+  gpio_init();
+  interrupt_init();
+  soft_timer_init();
+
+  CANHwSettings can_hw_settings = {
+    .tx = { GPIO_PORT_A, 12 },
```

Comment:
NIT: `.port` and `.pin` here. I don't have a preference between anonymous and GNU struct initialization but I think it should be consistent within a file which it isn't in this case.

Reply:
Yeah, fair enough. I do prefer to have designated initializers for my structs.